cognoml/tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,11 @@
+import os
+import mock
+from cognoml import utils 
+
+@mock.patch('os.mkdir')
+@mock.patch('os.path.exists')
```

Comment:
The value of the variable last passed to the function is stored in the mock_mkdir instance. This is what we are checking with the assertion that follows the function call. If we want to test successive calls, mock does have a easier way to do that using a slightly different syntax.